osbs/cli/main.py

```diff
@@ -704,16 +704,16 @@ def cli():
                               help="perform a scratch build")
     build_parser.add_argument("--yum-proxy", action='store', required=False,
                               help="set yum proxy to repos from koji/add-yum-repo params")
+    build_parser.add_argument('--release', action='store', required=False,
+                              help='release value to use')
+    build_parser.add_argument('--arrangement-version', action='store', required=False,
+                              help='version of inner template to use')
```

Comment:
I'm not entirely sure it makes sense to allow `--arrangement-version` for the build parse. We still support 'prod' (all-in-one) builds, which have no concept of a plugin arrangement.

We should at least disallow `--arrangement-version` if neither orchestrator nor worker build is being created.